data_prep/introspector.py

```diff
@@ -452,32 +453,46 @@ def get_project_funcs(project_name: str) -> Dict[str, List[Dict]]:
   return fuzz_target_funcs
 
 
+def _parse_arguments() -> argparse.Namespace:
+  """Parses command line args."""
+  parser = argparse.ArgumentParser(
+      description='Parse arguments to generate benchmarks')
```

Comment:
Nitpicking: For better consistency, could you please end the description with a period (e.g., `.`)? Thanks!
Same below.

Reply:
Sounds good, done!